scripts/rollup/bundles.js

```diff
@@ -85,7 +85,7 @@ const bundles = [
     moduleType: RENDERER,
     entry: 'react-dom',
     global: 'ReactDOM',
-    externals: ['react'],
+    externals: ['react', 'interaction-tracking'],
```

Comment:
As discussed let's make sure we're happy with the name before it becomes a public-visible dep

Reply:
Oh whoops, I didn't mean for this particular change to be in this PR. I'll move it to #13253